torch/fx/passes/backends/nvfuser/operator_support.py

```diff
@@ -164,7 +165,21 @@ def __init__(self):
             #     "_operator.truediv": None,
         }
 
-        super().__init__(support_dict)
+        # all aten ops that have a decomposition to prims.
+        ref_dict = {
+            # ===============================================================
+            # call_function builtins
+            # ===============================================================
+            "getattr": None,
+            "_operator.getitem": None,
+        }
+
+        # take the ops from the decomposition_table that were defined in torch._refs
```

Comment:
Can you tell me how come you have to filter out ops that weren't defined in torch._refs? If you are speculating the decomposition and then seeing if nvfuser supports the prims, it seems harmless to apply the non _refs decomps, since if they don't work they'll just map to ops nvfuser doesn't support and you're fine anyway. (And don't tell me that they might decompose too much: clearly you should STOP decomposing the moment you hit an aten function nvfuser says it knows how to handle directly!)